bot.js

```diff
@@ -372,3 +378,22 @@ function writeToTimerFile(channel){
 		channel.send(':thumbsup: Zakum has successfully modified the channel list.');
 	}
 }
+
+function listCommands(channel, isUserAdmin){
+	var helpMsg = '__Zakum Commands__ \n';
+
+	for(var i = 0; i < commands.length; i++){
```

Comment:
A cleaner way to do this and eliminate the nested for loop:
`
let helpMsg = '__Zakum Commands__ \n';
const commandsCanAccess = !isUserAdmin && commands.filter(command => command.admin === false)
commandsCanAccess.forEach(command => helpMsg += `**!${command.id}**  : ${command.description}` )
channel.send(helpMsg)
`